changelog.md

```diff
@@ -13,6 +13,16 @@ on task launch.
 If you rely on disk limits, you also need to configure Mesos appropriately. This includes configuring the correct
 isolator and enabling disk quotas enforcement with `--enforce_container_disk_quota`.
 
+#### New format for the `http_endpoints` parameter
+
+Changed the format of the `http_endpoints` parameters from a
+space-separated to a comma-separated list of endpoints, in order to be
+more consistent with the documentation and with the format used in other
+parameters.
+
+WARNING: if you are using multiple http endpoints, then you will have to
```

Comment:
You can also specify endpoints via the REST API and that's not affected.

So I would write something like: 

WARNING: If you use the http_endpoints parameter before with multiple space separated URLs, you will need to migrate to the comma-separated format.